packages/block-editor/src/components/block-list/block.js

```diff
@@ -454,14 +453,7 @@ function BlockListBlock( {
 	const blockMover = (
 		<BlockMover
 			clientIds={ clientId }
-			blockElementId={ blockElementId }
 			isHidden={ ! isSelected }
-			isDraggable={
```

Comment:
By removing the `isDraggable` prop we're no longer able to control the drag icon visibility. Is that ok? If it is, I wonder what this logic that was added is useful for:

> const isDragging = isDraggingBlocks && ( isSelected || isPartOfMultiSelection );

Reply:
It's handled directly in `BlockDraggable`

Reply:
Ah, just saw what `isDragging` is used for.

Reply:
This also makes the drag icon visible when we have a template lock (`isMovable`).

Reply:
I just saw that this PR moved [the visibility logic](https://github.com/WordPress/gutenberg/pull/18756/files?file-filters%5B%5D=.js&hide-deleted-files=true#diff-7768414d8422344fa32ea9aa3f3a9320R31) to the BlockDraggable component. Then:

- We should remove the `isMovable` logic from this component.
- I remain curious on whether the `isDraggable` prop is something that we need.

Reply:
> isDraggable

I think this is one of those things where we added it because "it might be useful" but in reality, no one knew about it and no one really used it. I have no proof but I'd say the opposite is also true. and even if it was used, the breakage is not really noticeable.

Reply:
Was commenting as I read the code and didn't see your responses. Fine to remove `isDraggable` if it's not used in practice.